pages/datarooms/index.tsx

```
@@ -119,58 +136,120 @@ export default function DataroomsPage() {
+                            </dt>
+                            <dd className="flex items-start gap-x-2">
+                              <div className="font-medium text-gray-900 dark:text-gray-200">
+                                {dataroom._count.views ?? 0}
+                              </div>
+                            </dd>
+                          </div>
+                        </dl>
+                      </CardContent>
+                    </Card>
+                  </Link>
+                ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)